frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -447,7 +453,10 @@ export const ExpandablePolicyCard = ({
         </Paragraph>
         <div className={classes.dropdownWrapper}>
           <LegacySelect
-            options={actionOptions}
+            options={actionOptions.map((option) => ({
+              ...option,
+              label: t(`policy_editor.action_${option.label}`),
```

Comment:
Here we can reuse the method from previous comment. :)
```suggestion
              label: getTranslationByActionId(option.label),
```

Reply:
ah, please see next comment instead, which is about handling the translation where the `actionOptions` is mapped.